content/general/concepts/github/github.md

```diff
@@ -0,0 +1,19 @@
+---
+Title: 'GitHub'
+Description: 'Github is a UI-based version control that uses Git.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'GitHub'
+  - 'Git'
+CatalogContent:
+  - 'learn-git'
+  - 'paths/learn-how-to-build-websites'
+---
+
+GitHub (GH) is a UI-based version control platform that runs on [Git](https://www.codecademy.com/resources/docs/git). It offers many services including version control,  static website hosting, open-source software ([OSS](https://www.codecademy.com/resources/docs/general/open-source)) projects, and much more.
+
+## Github Pages (GH Pages)
+
+Github pages is a service GH offers that will host static websites for free with an unlimited amount of vists. It automaticly has encryption with HTTPS. GH pages also supports cutsom domains.
```

Comment:
```suggestion
[Github Pages](https://pages.github.com/) is a service offered by GH for hosting static websites for free with an unlimited amount of visits. It automatically has encryption with HTTPS. GitHub Pages also supports custom domains.
```

Since not mentioned that many times in the entry, let's just go with the expanded "GitHub Pages" instead of the abbreviation "GH Pages".